### PR TITLE
Make updates

### PR DESCRIPTION
**Pull request by jlord**
This PR makes some updates to the project, mostly comments, geared towards making the parts of this Electron app more defined (i.e. renderer and main processes).

It also updates to use a newer version of Electron.

cc @kevinsawicki @zeke

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,8 +6,14 @@
...
     <h1>Hello World!</h1>
+    <!-- All of the Node.js APIs are available in this renderer process. -->
     We are using node <script>document.write(process.versions.node)</script>,
     Chromium <script>document.write(process.versions.chrome)</script>,
     and Electron <script>document.write(process.versions.electron)</script>.
   </body>
+
+  <script>
+    // You can also require other files to run in this process
+    require('./renderer.js')
```
Comment (kevinsawicki) [inline index.html]: Super minor, but the `.js` isn't required, if you think it makes it clearer I'd say leave it.

File: main.js [hunk 2]
```diff
@@ -49,3 +48,6 @@ app.on('activate', function () {
     createWindow();
   }
 });
+
+// In this file you can include the rest of your app specific main process
```
Comment (kevinsawicki) [inline main.js]: `app` -> `app's`

File: main.js [hunk 3]
```diff
@@ -49,3 +48,6 @@ app.on('activate', function () {
...
+// code. You can also put them in files and require them here.
```
Comment (kevinsawicki) [inline main.js]: Maybe `files` -> `separate files`?

File: index.html (hunk 1 above)
Comment (jlord) [inline index.html]: Yeah I think here we should be err on the side of being explicit 👍

**Comment by zeke** [comment]
🍦

**Pull request merged**